src/testController.ts

```diff
@@ -106,8 +110,14 @@ export class TestController {
 
       // Add test methods as children to the test class so it appears nested in Test explorer
       // and running the test class will run all of the test methods
-      if (name.startsWith("test_")) {
-        testItem.tags = [this.debugTag];
+
+      // Older Ruby LSP versions may not include 'kind' so we try infer it from the name.
+      if (!res.data?.kind && name.startsWith("test_")) {
+        res.data.kind = "example";
+      }
```

Comment:
Can we move this part to the `else` branch of the above `if (res.data?.kind) {` condition, and insert an `example` tag there?